wls-common/security/src/test/java/de/muenchen/oss/wahllokalsystem/wls/common/security/BezirkIDPermissionEvaluatorImplTest.java

```diff
@@ -0,0 +1,59 @@
+package de.muenchen.oss.wahllokalsystem.wls.common.security;
+
+import de.muenchen.oss.wahllokalsystem.wls.common.security.testultils.LoggerExtension;
+import java.util.HashMap;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+class BezirkIDPermissionEvaluatorImplTest {
+
+    private final BezirkIDPermissionEvaluatorImpl unitUnderTest = new BezirkIDPermissionEvaluatorImpl();
+
+    @Nested
+    class TestTokenUserBezirkIdMatches {
+
+        @RegisterExtension
+        public LoggerExtension loggerExtension = new LoggerExtension();
+
+        @Test
+        void warnOnAuthenticationIsNull() {
+            unitUnderTest.tokenUserBezirkIdMatches("1234", null);
+            Assertions.assertThat(loggerExtension.getFormattedMessages().contains("No authentication object for bezirkId=1234")).isTrue();
```

Comment:
macht es aus meiner Sicht lesbarer

```suggestion
            Assertions.assertThat(loggerExtension.getFormattedMessages()).contains("No authentication object for bezirkId=1234");
```